launchUI.py

```diff
@@ -30,6 +30,7 @@
 MEM_WIDTH = 50
 CONFIG_MEM_SIZE = 8
 DATA_MEM_SIZE = 4
+thickness = 1

```

Comment:
Plz rename this as `HIGHLIGHT_THICKNESS`.

Reply:
okay I will do it.